wiki/en/en-Sound-Devices.md

```diff
@@ -1,87 +1,138 @@
+This page lists (thanks to Jamulus users) the audio devices known to work (or not) with Jamulus.  
+_**Note**: depends on how the device works on your system (Linux, MacOS, Windows, etc.)_.
 
-### Known to Work with Jamulus
+You can update it with the information you have. **Please indicate the platform(s) you are using!**.
 
-#### USB Microphones
+## Audio devices known to work with Jamulus
 
-* Blue Yeti microphone. Works well on Mac (there is a checkbox to turn off local monitoring). Works OK on Windows with ASIO4all but with a bit more latency than on Mac. You can turn off the local monitoring if you go deep into the settings. Latency on Linux is good but I did not find any way to turn off the local monitoring.
+### USB/USB-C devices
 
-* [Shure X2u](https://www.shure.com/en-GB/products/accessories/x2u-xlr-usb-interface) XLR to USB audio interface with headphone jack, converting an XLR mic into a USB mic. Set PC/Mic mix dial to 100% PC and listen to the output on headphones. Tested using Raspberry Pi 4 running Raspberry Pi OS Buster, kernel version 4.19.
+#### Microphones / DI Boxes
 
+* **[Blue Yeti](https://www.bluemic.com/fr-fr/products/yeti/)**, USB Microphone  
+Works well under **macOS** (there is a check option to disable local monitoring).  
+Works well under **Windows 10 with ASIO4All**, a bit more latency than under macOS. You can disable local monitoring by going deep into the settings.  
+Latency is good under **Linux**. You can't disable local monitoring (or we haven't found out how to do it, feel free to send us the information if you know how to do it).
 
-#### Audio interfaces
+* **[Shure X2u](https://www.shure.com/en-GB/products/accessories/x2u-xlr-usb-interface)**, USB to XLR audio interface with headphone jack connector that converts a wired microphone to a USB microphone.  
+Set the PC/Mix dialog box to 100% PC to listen to Jamulus' mix on the headphone output.  
+Tested with a Raspberry Pi 4 under **Raspberry Pi OS Buster**, kernel version 4.19.
 
-* Ammoon [AGM02](https://www.ammoon.com/p-i3974.html) and [AGM04](https://www.ammoon.com/p-i4049.html) USB mixer: relatively inexpensive 4 channel USB mixers, both tested on MacOS Catalina, the AGM02 tested on Windows 10. On Windows 10, with ASIO4ALL, the AGM02 can be used as an input, and the onboard audio as the output. There is no way to monitor only the USB return signal on the AGM02.  (USB Audio)
-**Note:** The Ammoon AGM04 appears to be a re-branded [ART USBMix4](https://artproaudio.com/product/usbmix4-four-channel-mixer-usb-audio-interface/).
+* **[Behringer UGC102](https://www.behringer.com/behringer/product?modelCode=P0198)**, Guitar-to-USB Interface  (guitar and bass) 
+_Note: On **macOS**, appears as "USB Audio Codec". When selected for input and output in Jamulus, the **UGC102** headphone jack is the output (and cannot be used for input)._
```

Comment:
Not a sure, but shouldn't it be "On macOS this device appears as..."

Reply:
Fixed